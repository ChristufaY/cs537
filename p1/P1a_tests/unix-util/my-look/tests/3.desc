sample test: first arg only
