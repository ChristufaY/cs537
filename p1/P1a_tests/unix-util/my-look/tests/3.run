./my-look -V -h -f file.txt
