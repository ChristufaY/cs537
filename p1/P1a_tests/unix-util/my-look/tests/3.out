my-look from CS537 Spring 2022
